internal/pkg/cli/env_init.go

```diff
@@ -300,6 +326,93 @@ func (o *initEnvOpts) askCustomizedResources() error {
 	if o.NoCustomResources {
 		return nil
 	}
+	adjustOrImport, err := o.prompt.SelectOne(
+		envInitDefaultEnvConfirmPrompt, "",
+		envInitCustomizedEnvTypes)
+	if err != nil {
+		return fmt.Errorf("select adjusting or importing resources: %w", err)
+	}
+	switch adjustOrImport {
+	case envInitImportEnvResources.String():
```

Comment:
could we get rid of enum and just use `case envInitImportEnvResourcesSelectOption:`?